api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPTEligiblePatientListCohortQueries.java

```diff
@@ -1509,12 +1121,11 @@ public CohortDefinition getIPTB1part2() {
             + "UNION "
             + "SELECT p.patient_id FROM patient p INNER JOIN encounter e ON p.patient_id = e.patient_id "
             + "                                     INNER JOIN obs o ON e.encounter_id = o.encounter_id "
-            + "                                     INNER JOIN obs o2 ON e.encounter_id = o2.encounter_id "
-            + "WHERE p.voided =0 AND e.voided = 0 AND o.voided=0 AND o2.voided=0 "
+            + "WHERE p.voided =0 AND e.voided = 0 AND o.voided=0 "
             + "  AND e.encounter_type = ${53} "
             + "  AND o.concept_id = ${23985} AND o.value_coded = ${656} "
-            + "  AND o2.concept_id = ${6129} AND o2.value_datetime IS NOT NULL "
```

Comment:
The INH must be checked using the new TPT structure. Value_coded = Fim (id= 1267) for concept_id = 165308